api/service.go

```diff
@@ -0,0 +1,111 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/uuid"
+	ds "github.com/ipfs/go-datastore"
+	core "github.com/textileio/go-textile-core/store"
+	tserv "github.com/textileio/go-textile-core/threadservice"
+	pb "github.com/textileio/go-textile-threads/api/pb"
+	es "github.com/textileio/go-textile-threads/eventstore"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type service struct {
+	datastore ds.Datastore
+	threads   tserv.Threadservice
+
+	stores map[string]*es.Store
```

Comment:
Didn't put much thought into this... just a dirty way to access a created store.